uwblueprint/community-fridge-kw
# fix: update seeding script with donors, volunteers, checkin, content

## Brief description. What is this change? 

### [Fix seeding script](https://www.notion.so/uwblueprintexecs/Update-seeder-3407659d353c4ef5b3592585940e84f3)

Important changes:
- Fixed the auth uid issue so that we can have a verified donors, volunteer, and admin test account locally (instead of going through the whole sign up flow) 🥳 
- Added in the content seed data to easily use instead of creating each time
- Updates to the other tables

![Screen Shot 2022-03-14 at 8 02 37 PM](https://user-images.githubusercontent.com/19617248/158281823-d4cc79b1-b49d-4861-86ae-dadc340d5c56.png)
<img width="553" alt="Screen Shot 2022-03-14 at 8 03 43 PM" src="https://user-images.githubusercontent.com/19617248/158281826-36e92109-d81c-4279-a032-ec53c47850a6.png">


## Steps to test
1. Update your secrets in `.env`
2. Follow the seeding script setup to run your container and `node seed up` 
3. Verify you can log in with the new users
